libraries/chaindb/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Defines ChainBase library target.
+project( ChainBase )
+
+file(GLOB HEADERS "include/chainbase/*.hpp")
+add_library( chainbase src/chainbase.cpp src/pinnable_mapped_file.cpp ${HEADERS} )
+target_link_libraries( chainbase PUBLIC pthread Boost::system )
```

Comment:
Instead of `pthread`, the portable way is to `Threads::Threads`. We also (I think) consistently set up options on `Threads::Threads` with:
https://github.com/AntelopeIO/spring/blob/504fa82dca9172a36056a67dc1932e063bdb6adf/libraries/libfc/CMakeLists.txt#L6-L8
(the first one, `CMAKE_THREAD_PREFER_PTHREAD`, is no longer documented, dunno what's up with that)

Reply:
https://github.com/AntelopeIO/spring/pull/1449/commits/9516ea20b3f2ef5d493cf6d1d09ed0a8a1aee761